Code/Source/Lidar/ROS2LidarSensorComponent.cpp

```diff
@@ -55,6 +59,29 @@ namespace ROS2
         m_sensorConfiguration.m_publishersConfigurations.insert(AZStd::make_pair(type, pc));
     }
 
+    void ROS2LidarSensorComponent::Visualise()
+    {
+        if (m_visualisationPoints.empty())
+        {
+            return;
+        }
+
+        auto defaultScene = AZ::RPI::Scene::GetSceneForEntityId(GetEntityId());
```

Comment:
Do we need to fetch the current scene every `Visualise()`? 
Maybe move it to `Activate()`.